bin/nest_rmu_linear.csh

```diff
@@ -0,0 +1,165 @@
+#!/bin/csh
+# Create rmu_nest files with variant e-folding time 
+set echo
+set time = 1
+#
+
+setenv TOOLS /home/sm_alfal/sea/TOPAZ5/NERSC-HYCOM-CICE/hycom/hycom_ALL/hycom_2.2.72_ALL
```

Comment:
Excuse my ignorance of our own systems, but how is it done with folders now? Do we use environment variables or do we edit them by hand in each script?